cmake/FindOpenVDB.cmake

```diff
@@ -306,9 +306,6 @@ list(APPEND _OPENVDB_LIBRARYDIR_SEARCH_DIRS
 set(_OPENVDB_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
 
 set(OPENVDB_PYTHON_PATH_SUFFIXES
-  ${CMAKE_INSTALL_LIBDIR}/python
```

Comment:
I've replaced this patterns with call to `file(GLOB)` since I would need to add in my case

```cmake
${CMAKE_INSTALL_LIBDIR}/python3.8/dist-packages
${CMAKE_INSTALL_LIBDIR}/python3.8/site-packages
# ... and many more depending on the pyhton version
```

I realized that it's better to let cmake find in your local system what's in there. The `libxx/pytonx` pattern might be removed at all, but I'm not sure why this is there in first place